packages/@office-iss/react-native-win32/src/Libraries/Components/View/Tests/ViewWin32Test.tsx

```diff
@@ -245,6 +247,20 @@ const CursorExample: React.FunctionComponent = () => {
   );
 }
 
+const EnableFocusRingExample: React.FunctionComponent<{}> = () => {
+  return (
```

Comment:
nit test example that switches between the two states?

Reply:
The first two tests provide examples for enableFocusRing being set to false, but I can add an explicit false state.